apiserver/controllers/controllers.go

```diff
@@ -170,6 +215,23 @@ func (a *APIController) NotFoundHandler(w http.ResponseWriter, r *http.Request)
 	}
 }
 
+func (a *APIController) MetricsTokenHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+
+	if !auth.IsAdmin(ctx) {
+		handleError(w, gErrors.ErrUnauthorized)
+		return
+	}
+
+	token, err := a.auth.GetJWTMetricsToken(ctx)
+	if err != nil {
+		handleError(w, err)
+		return
+	}
+	w.Header().Set("Content-Type", "application/json")
+	json.NewEncoder(w).Encode(runnerParams.JWTResponse{Token: token})
```

Comment:
It's complaining that we're throwing away the error here. I put a ```log.Printf``` everywhere else, to see when such an error occurs, but feel free to add ```// nolint``` at the end of the line.